formlayout.py

```diff
@@ -334,15 +334,32 @@ class PushLayout(QHBoxLayout):
     """Push buttons horizontal layout"""
     def __init__(self, buttons, parent=None):
         QHBoxLayout.__init__(self)
+        self.result = parent.result
+        self.dialog = parent.get_dialog()
         for button in buttons:
             label, callback = button
             self.btn = QPushButton(label)
             if SIGNAL is None:
-                self.btn.clicked.connect(callback)
+                self.btn.clicked.connect(self.call(callback))
```

Comment:
I haven't tested it but why not directly write this:
```diff
-                self.btn.clicked.connect(self.call(callback))
+                self.btn.clicked.connect(lambda: self.apply(callback))
```

...in order to avoid having to define the ``call`` method:
```diff
-    def call(self, callback):
-        return lambda: self.apply(callback)
```

Other than that, the PR seems ok to me.